ci/common/suite.sh

```diff
@@ -11,14 +11,37 @@ FAIL_SUMMARY=""
 END_MARKER="$BUILD_DIR/.tests_finished"
 FAIL_SUMMARY_FILE="$BUILD_DIR/.test_errors"
 
+ANSI_CLEAR="\033[0K"
+
+travis_fold() {
+  local action="$1"
+  local name="$2"
+  name="$(echo -n "$name" | tr '\n\0' '--' | sed -r 's/[^A-Za-z0-9]+/-/g')"
+  name="$(echo -n "$name" | sed -r 's/-$//')"
```

Comment:
@jamessan Does this “Or” mean that `\+` not supported? Or this is the third alternative?